### DISCUSSION TITLE
Dash to panel broken in Ubuntu 24.04?

### DISCUSSION THREAD

**Discussion by ad203amiar** [home-sweet-gnome/dash-to-panel#2100] [Q&A] [answered]
When i installed Dash to panel and wanted to adjust it, the Ubuntu Dock and dash to panel both showed they had problens in Extension Manager. Anyone else tried it on Ubuntu 24.04? does it work?

**Comment by charlesg99** [comment, marked as answer]
Yes, the way dash to panel disables ubuntu dock when initialized was broken in the latest gnome-shell version. Till a fix for that is released, the best way to solve the issue is to disable the ubuntu dock completely using `gnome-extensions disable ubuntu-dock@ubuntu.com` or a program like extension manager.

**Reply by ad203amiar** [reply]
ok, thank you.
